styles/signals.mapcss

```diff
@@ -1627,11 +1678,13 @@ node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:main"="AT-
 	allow-overlap: true;
 }
 
-/*************************/
-/* FI main light signals */
-/* general rules         */
-/*************************/
-node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:main"=~/^FI:Po(-v)?$/]["railway:signal:main:form"=light]
+/*****************************/
+/* FI main light signals     */
+/* FI combined block signals */
+/* general rules             */
+/*****************************/
+node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:main"=~/^FI:Po(-v)?$/]["railway:signal:main:form"=light],
```

Comment:
I think this section should be "inverted": for all 3 signals use only the matching rules below that match exactly one signal. End every of them using "set .fi-main-combined;" (or however you want to call that tag). Then move this section below the 3 others and change it to match "node.fi-main-combined", with the style rules as above. This will avoid doing the complete matches twice, and it will avoid the need for a regular expression match.